### PR TITLE
fix databricks environment with raiwidgets dashboard

### PR DESCRIPTION
**Pull request by imatiach-msft**
fixed raiwidgets to work in databricks environment

image

### CODE REVIEW

**Comment by imatiach-msft** [comment]
fixes issue https://github.com/microsoft/responsible-ai-widgets/issues/400

File: raiwidgets/raiwidgets/dashboard.py [hunk 1]
```diff
@@ -76,7 +77,10 @@ def __init__(self, *,
 
         html = self.load_index()
         print(f'{dashboard_type} started at {self._service.env.base_url}')
-        display(HTML(html), metadata=dict(isolated=True))
+        if "DATABRICKS_RUNTIME_VERSION" in os.environ:
```
Comment (romanlutz) [inline raiwidgets/raiwidgets/dashboard.py:80]: Shouldn't this be a function in the flask wrapper? I think it's actually there already, we just don't use it yet.

Comment (imatiach-msft) [reply raiwidgets/raiwidgets/dashboard.py]: done, used display function

File: raiwidgets/raiwidgets/dashboard.py [hunk 2]
```diff
@@ -9,7 +9,8 @@
 import os
 from html.parser import HTMLParser
 import uuid
-from .explanation_constants import ExplanationDashboardInterface
+from .explanation_constants import (
+    ExplanationDashboardInterface, DatabricksInterfaceConstants)
```
Comment (xuke444) [inline raiwidgets/raiwidgets/dashboard.py:13]: seems this new import is not used?

Comment (imatiach-msft) [reply raiwidgets/raiwidgets/dashboard.py:13]: fixed

**Review by xuke444 — APPROVED** [review]

**Comment by jomedinag** [comment]
Did the changes go through? Apparently the commit failed

image

**Comment by romanlutz** [comment]
@jomedinag the notebook tests were temporarily not running so we bypassed them for this PR (and another one). They're back up now. This didn't affect anything, though, since there were no changes to notebooks in this PR. To actually get the changes into the package we have to do another release first. @imatiach-msft any plans for that?

**Pull request merged**